apps/src/templates/Notification.jsx

```diff
@@ -322,9 +322,9 @@ const styles = {
       backgroundColor: color.teal
     },
     [NotificationType.bullhorn]: {
-      borderColor: color.teal,
-      color: color.teal,
-      backgroundColor: color.teal
+      borderColor: color.yellow,
+      color: color.yellow,
+      backgroundColor: color.yellow
```

Comment:
Just double checking: Do we want this updated for all bullhorn notifications?

Reply:
good catch, I'll check on that

Reply:
@TurnerRiley do you have context on whether this should have been for all bullhorn notifications or just these new ones?

Reply:
I added a new notification type "bullhorn_yellow" and used that for the one bullhorn alert we are using. We don't seem to currently be using the bullhorn elsewhere, but it can be enabled [via props](https://github.com/code-dot-org/code-dot-org/blob/44395eb2105182e13b77ff1e4144003687b1e59f/apps/src/templates/census2017/YourSchool.jsx#L90), so changing it to yellow everywhere could cause unexpected issues in the future.

Reply:
Oh yes I think just the new ones? But also it's been so long since I made this PR so I do not remember the exacct context. Creating the new notification type sounds great!